install-steps.md

```diff
@@ -539,7 +538,34 @@ sudo cp ./oc /usr/local/bin/oc
 oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
 ~~~
 
+### Create RHCOS images cache (Optional)
 
+The installer needs to download two images, the RHCOS image used by the bootrstrap VM and the RHCOS image used by the installer to provision the different nodes.
```

Comment:
bootstrap spelled incorrectly

Reply:
I'd also had context as to why someone would want RHCOS image cacheing. Here it just says you can use it but add the why.

Reply:
Done.